### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by juancho11gm**

### CODE REVIEW

File: 3-semantic-html/index.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+      <!-- nav content -->
+      </nav>
+   </header>
+   
+  <footer>
+     <!-- footer content -->
+    <p> PushDev 2020 </p>
+  </footer>
+```
+
```
Comment (pushdev-code) [inline 3-semantic-html/index.md:49]: Please include Document Structure, Textual meaning and Media type from https://html.com/semantic-markup/

File: 3-semantic-html/index.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+Build in HTML a simple form with semantic structure. The form must have the following elements:
+
+1. First name
+2. Last name
+3. Email-Address
+4. Date of birth
+5. Gender (Using select)
+6. Socioeconomic status (Using radiobutton)
+7. IMG (Searching file
```
Comment (pushdev-code) [inline 3-semantic-html/index.md:60]: let's keep this exercise for the forms section. Let's create an example of a semantic markup as exercise. I suggest this image https://images.thoughtbot.com/cp-design-for-the-web/HLyRTWdhTsaa2JtqtzHK_layout-html5.png  and in the `<main>` include an article like this: 
```html
<article>
    <header>
      <h1>The Robot Uprising and You</h1>
    </header>
    <section>
      <p>
        ...
      </p>
    </section>
    <footer>
      Posted by Ralph Bot on August 29th
    </footer>
  </article>
```

File: 3-semantic-html/index.md (hunk 1 above)
Comment (juancho11gm) [reply 3-semantic-html/index.md:131]: done

File: 3-semantic-html/index.md (hunk 2 above)
Comment (juancho11gm) [reply 3-semantic-html/index.md]: I posted an image of the DOM  in this example.

**Pull request merged**